app/metrics/CloudWatch.scala

```diff
@@ -37,22 +41,11 @@ object CloudWatch {
     acwac
   }
 
-  val prodStage = "PROD"
-  val allStacks = "*"
-  val namespace = "AMIs"
+  val defaultNamespace = s"AMIable-$stage"
 
-  private val dimensions = List(
-    new Dimension()
-      .withName("stage")
-      .withValue(prodStage),
-    new Dimension()
-      .withName("stack")
-      .withValue(allStacks)
-  )
-
-  private[metrics] def putRequest(metricName: String, value: Int): PutMetricDataRequest = {
+  private[metrics] def putRequest(metricName: String, value: Int, dimensions: List[Dimension] = List.empty, namespace: String = defaultNamespace): PutMetricDataRequest = {
     new PutMetricDataRequest()
-      .withNamespace(namespace)
```

Comment:
@philmcmahon I noticed that the PROD metrics are still coming through to the AMIable-PROD namespace. Am I reading this right, is putRequest ignoring the namespace argument and just using defaultNamespace? Which would justify what's happening?